src/components/landing/ListingDetails.js

```diff
@@ -0,0 +1,72 @@
+import Image from "next/image";
+import Link from "next/link";
+import EmptyListing from "./EmptyListing";
+import { useEffect, useState } from "react";
+import Loading from "../common/Loading";
+
+const ListingDetails = ({ listing }) => {
+  const LOGO_WIDTH = 20;
+  const LOGO_HEIGHT = 20;
+  const ICON_SRC = "/octicon_goal-24.png";
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    if (listing) {
+      setIsLoading(false);
+    }
+  }, [listing]);
+
+  return (
+    <>
+      {isLoading ? (
+        <Loading />
+      ) : listing ? (
+        <main>
+          <section className="p-10">
+            <article className="w-2/3 pb-5  lg:my-0 lg:px-12 ">
+              <h1 className="font-medium text-xl py-1">Our Community</h1>
+              <p>{listing.community_description}</p>
+              <p>
+                <Link href={listing.community_url} className="text-primary">
```

Comment:
@Achele please remove lines `31` to `35`. It is too specific. What if the community is for UI/UX Designers? Will they still be unlocking their potentials as developers?